.crowdin.yml

```
@@ -0,0 +1,3 @@
+files:
+  - source: /locales/base.json
+    translation: /locales/%two_letters_code%.json
```

So it's recognized with the right type in Crowdin (allows pluralization, though of course it doesn't seem we need that here, but doesn't hurt).
```suggestion
    translation: /locales/%two_letters_code%.json
    type: i18next_json
```